corehq/apps/data_cleaning/templates/data_cleaning/tables/table_with_controls.html

```diff
@@ -72,7 +72,7 @@
           @click="$dispatch('updateUndoSummaryMessage');"
         >
           <i class="fa-solid fa-undo"></i>
-          {% trans "Undo" %}
+          {% trans "Undo" %} {# note: undo for multiple records #}
```

Comment:
@mjriley how is adding a comment for differentiating? that's the best I can think if since the show condition is alpine, not a template variable

Reply:
One other thought for differentiation would be to give each of these elements an id.